trigger/utils/templates.py

```diff
@@ -0,0 +1,76 @@
+#coding=utf-8
+
+"""
+Templating functions for unstructured CLI output.
+"""
+
+__author__ = 'Thomas Cuthbert'
+__maintainer__ = 'Thomas Cuthbert'
+__email__ = 'tcuthbert90@gmail.com'
+__copyright__ = 'Copyright 2016 Trigger Org'
+
+
+import sys
+import os
+from trigger.conf import settings
+from twisted.python import log
+
+try:
+    import textfsm
+except ImportError:
+    print("""
+    Woops, looks like you're missing the textfsm library.
+
+    Try installing it like this::
+
+        >>> pip install gtextfsm
+    """)
+
+
+# Exports
+__all__ = ('get_template_path', 'load_cmd_template', 'get_textfsm_object')
+
+
+def get_template_path(cmd, dev_type=None):
+    """
+    Return textfsm templates from the directory pointed to by the TEXTFSM_TEMPLATE_DIR trigger variable.
+
+    :param dev_type: Type of device ie cisco_ios, arista_eos
+    :type  dev_type: str
+    :param cmd: CLI command to load template.
+    :type  cmd: str
+    :returns: String template path
+    """
+    t_dir = settings.TEXTFSM_TEMPLATE_DIR
+    return '{0}/{1}_{2}.template'.format(t_dir, dev_type, cmd.replace(' ', '_')) or None
```

Comment:
Can you please use `os.path.join()` here vs. `{0}/{1}` for `(t_dir, dev_type)`. I'm imagining a case where the value to `TEXTFSM_TEMPLATE_DIR` contains a trailing slash and causes things to fail strangely.